src/AaveV3StrategicAssets_20230606/libs/VersionedInitializable.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
```

Comment:
perhaps better to use https://github.com/bgd-labs/solidity-utils/blob/main/src/contracts/transparent-proxy/Initializable.sol#L65C19-L65C32 - with this you no longer have to initialize the impl

Reply:
@sakulstra not sure how to use this one/do upgrades afterwards 😕

Reply:
it's very similar, you just put the `reinitializer(uint8 version)` modifier on a function https://github.com/bgd-labs/solidity-utils/blob/main/src/contracts/transparent-proxy/Initializable.sol#L35